src/Abp/Runtime/Caching/AbpCacheBase.cs

```diff
@@ -67,7 +67,7 @@ public virtual TValue Get(TKey key, Func<TKey, TValue> factory)
 
                         try
                         {
-                            Set(key, item);
+                            Set(key, item, DefaultSlidingExpireTime, DefaultAbsoluteExpireTime);
```

Comment:
These changes are unnecessary since the `Set` functions already use the default.

From the docstring of `IAbpCache<TKey, TValue>`'s `Set` method:

```c#
/// Use one of the expire times at most (<paramref name="slidingExpireTime"/> or <paramref name="absoluteExpireTime"/>).
/// If none of them is specified, then
/// <see cref="ICacheOptions.DefaultAbsoluteExpireTime"/> will be used if it's not null. Othewise, <see cref="ICacheOptions.DefaultSlidingExpireTime"/>
/// will be used.
```